open-vlc.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+#
+# This wrapper makes a lot of assumptions on how openscad is called
+# from within FreeCAD
+#
+
+exec flatpak-spawn --directory="$(pwd)" --host flatpak run --filesystem="$XDG_CACHE_HOME/tmp" org.videolan.VLC "$@"
```

Comment:
So it does not work with a native installed VLC

Reply:
why would you do that?

Reply:
I'm sorry, but I'm not sure what I should do here… should I call VLC in a different way?  If so, what is the proper way to call an external flatpak/tool outside the current flatpak?

I came across https://github.com/flatpak/flatpak/issues/4046#issuecomment-1177381023 and figured that `flatpak-spawn` was the (semi-?)official way to do this.

I'm happy to use a different way, but I don't know how to do this differently.  As far as I'm concerned it's fine if this only works with a flatpaked VLC; someone who doesn't use a flatpaked VLC would likely also not use a flatpaked MediathekView.  MediathakView upstream provides an AppImage download for those who don't use flatpak so I don't think this flatpak needs to care for semi-flatpaked installations 🤔

Reply:
I was replying to "does not work with a native installed VLC".

Calling the flaptak version is the right approach.

Reply:
Ah, sorry, then I misunderstood your remark 😊 So I can consider this conversation resolved?

Reply:
Why `--filesystem="$XDG_CACHE_HOME/tmp"` is needed here?

Reply:
I don't know. I copied this from the above link.

I'll try to remove it, and see if it still works 😇

Reply:
> I don't know. I copied this from the above link.

please don't copy stuff you don't understand.

Reply:
@Erick555 I've removed the redundant argument.

> please don't copy stuff you don't understand.

I appreciate the lecture but come on, if I only copied what I fully understand I'd give you an empty pull request.  I don't even understand YAML in all its weird weirdness 😉

I wanted to get this working before adding the polish, and that this use of spawn to start other flatpaks doesn't seem to be documented anywhere didn't exactly help with understanding 😉   I'm sorry that I forgot to cleanup this script before submitting, tho.